gradlew

```diff
@@ -78,7 +78,7 @@ location of your Java installation."
     fi
 else
     JAVACMD="java"
-    which java >/dev/null 2>&1 || die "ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
+    which kotlin >/dev/null 2>&1 || die "ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
```

Comment:
Why java is changed by kotlin in gradle wrapper?

Reply:
Seems like this is an automatic thing done by Android Studio after adding Kotlin support to at least 1 of the project modules. It's also probably necessary to compile Kotlin properly, though I'm not really sure.

Reply:
Yeah but kotlin has nothing to do in gradlew, gradlew is gradle wrapper, the utility that download gradle, and gradle is the utility to download kotlin.

Please undo that, as this will break `./gradlew` command line on linux and probably JitPack too.

Reply:
Thanks for the clarification! I reverted the changes and will mark the conversation as resolved.

Reply:
Glad you take my comment into account :)